core/src/main/java/com/netflix/conductor/service/ExecutionService.java

```diff
@@ -386,23 +386,26 @@ public void archiveWorkflow(String workflowId, boolean retainState, boolean inde
         executionDAOFacade.archiveWorkflow(workflowId, retainState, indexWorkflow);
     }
 
+    public SearchResult<WorkflowSummary> search(
+            String query, String freeText, int start, int size, List<String> sortOptions) {
+
```

Comment:
Since we have customized search() method (as part of ISICO-13954) to remove WF from indexDB when WF is not found in transactionalDB(Redis). We need to retain our changes.

Reply:
Updated the changes.